src/utils/sort-versions.ts

```diff
@@ -33,14 +59,15 @@ function compare(a: semver.SemVer | string, b: semver.SemVer | string) {
  * @returns {RunnableVersion[]}
  */
 export function sortVersions(versions: RunnableVersion[]): RunnableVersion[] {
-  type VerSemRun = [
-    ver: string,
-    sem: semver.SemVer | null,
-    run: RunnableVersion,
-  ];
-  const sorted = versions
-    .map((run): VerSemRun => [run.version, semver.parse(run.version), run])
-    .sort(([vera, sema], [verb, semb]) => compare(sema || vera, semb || verb));
-  sorted.forEach(([_1, _2, run], idx) => (versions[idx] = run));
-  return versions;
+  type VerSemRun = [ver: string, run: RunnableVersion];
+
+  const filtered = versions.filter((runnable) =>
+    semver.valid(runnable.version),
+  );
```

Comment:
`semver.valid()` is literally just a boolean wrapper around semver.parse() so this is another single-time-use parse